src/common/src/lib.rs

```diff
@@ -44,6 +44,7 @@
 #![feature(negative_impls)]
 #![feature(bound_map)]
 #![feature(array_methods)]
+#![feature(btree_cursors)]
```

Comment:
```suggestion
```

Reply:
This is still needed by `Sentinelled`

Reply:
This is still needed by `EstimatedBTreeMap`.